packages/backend/src/services/GdriveService/GdriveService.ts

```diff
@@ -68,6 +69,23 @@ export class GdriveService extends BaseService {
             throw new ForbiddenError();
         }
 
+        if (
```

Comment:
is this also true for schedules?

Reply:
this seemed to be the only one that receives a metricquery from the API instead of a dashboard/chart uuid.